target/linux/ipq806x/config-4.9

```diff
@@ -313,6 +313,8 @@ CONFIG_MTD_M25P80=y
 CONFIG_MTD_NAND=y
 CONFIG_MTD_NAND_ECC=y
 CONFIG_MTD_NAND_QCOM=y
+CONFIG_MTD_SPINAND_MT29F=y
+CONFIG_MTD_SPINAND_ONDIEECC=y
```

Comment:
Did you add that manually?

Reply:
Sorry, I don't understand what do you mean by that?

Reply:
You didn't use `make kernel_menuconfig CONFIG_TARGET=target` and put these lines by hand instead, am I right? :)

Reply:
Yes, that is correct.

Reply:
> Yes, that is correct.

Then it's wrong, you broke alphabetical order here.

Reply:
@pepe2k Thank you,I did not know how to use make kernel_menuconfig CONFIG_TARGET=target so far.
Now it should be correct